conf/deployment/aws/ipi_1az_rhcos_lso_3m_3w.yaml

```diff
@@ -13,7 +13,7 @@ ENV_DATA:
   worker_replicas: 3
   master_replicas: 3
   worker_instance_type: 'i3en.2xlarge'
-  device_size: 2300  # Value pulled from here: https://bugzilla.redhat.com/show_bug.cgi?id=1812557#c27
+  device_size: 2328
```

Comment:
@clacroix12 I am afraid that this value is getting overwritten by cli parameter in our jenkins job https://ocs4-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/qe-deploy-ocs-cluster/9240 I see storageCluster has 2Ti. http://magna002.ceph.redhat.com/ocsci-jenkins/openshift-clusters/jnk-ai3cl33-t1/jnk-ai3cl33-t1_20200629T085616/logs/failed_testcase_ocs_logs_1593421657/deployment_ocs_logs/ocs_must_gather/quay-io-rhceph-dev-ocs-must-gather-sha256-4db6344cc517f57df8ca261314aeba25d5348d3f033a14e90267b75b905efc8a/storagecluster.yaml

Reply:
Alright, the way I see it we have two options. We can change the behavior of the `OSD_SIZE` param in jenkins to default to an empty string in the case of our LSO trigger jobs. This has the downside of not being caught if someone is deploying a one-off LSO deployment without using the trigger job though. The other way is we can change the default behavior to not set a value in jenkins (default to empty string) and rely on the default in ocs-ci which can be set to 2Ti (overwritten by the config value). WDYT @petr-balogh ?

Reply:
We currently do report of this value in our spreadsheet, it it will be empty we will not report that value there I guess. So we need to align the job to report some value like DEFAULT if not specified.

We can for AWS LSO jobs to use empty value in param - or we need to define this size 2328 in our OSD_SIZE parameter and propagate this value from LSO jobs as default one.   I am for second option to have it in the report as well.

Reply:
@petr-balogh https://gitlab.cee.redhat.com/ocs/ocs4-jenkins/-/merge_requests/294